src/collapse/guess.rs

```diff
@@ -120,6 +120,6 @@ impl Collapse for Folder {
     }
```

Comment:
I assume this change was necessary for fuzzing, and doesn't disrupt normal functionality.

Reply:
yes this was interesting. they are implementing a public interface, but directly saying it's unreachable and should panic. in my fuzz target, I call each public interface of the trait, so this was giving false positive crashes. It could be changed to

```rs
if cfg!(fuzzing) { None } else { unreachable!() }
```